File: c++/leetcode/168. Excel Sheet Column Title.cpp
class Solution
{
public:
  string convertToTitle(int n)
  {
    string ans{};

    while (n)
    {
      ans = static_cast<char>((--n % 26) + 'A') + ans;
      n /= 26;
    }

    return ans;
  }
};

File: javascript/leetcode/168. Excel Sheet Column Title.js
/**
 * @param {number} n
 * @return {string}
 */
var convertToTitle = function (n) {
  let ans = "";

  while (n) {
    ans = String.fromCharCode((--n % 26) + "A".charCodeAt(0)) + ans;
    n = parseInt(n / 26);
  }

  return ans;
};
